app/assets/javascripts/views/shared/chartWidgetView.js

```diff
@@ -253,7 +255,36 @@
       // We append the button to the DOM
       this.el.querySelector('.js-switch-button').appendChild(button);
     },
+    /**
+     * Render the Toggle Visibility button and attach an event listener to it
+     */
+    _renderToggleVisibilityButton: function () {
+      // We create the button
+      var button = document.createElement('button');
+      button.type = 'button';
+      button.classList.add('c-button', 'toggle-visibility-button');
+      button.textContent = 'Hide';
+      // We attach the listener
+      button.addEventListener('click', () => {
```

Comment:
Avoid using ES6 syntax (i.e. arrow functions) cause Backbone projects are not transpiled.